docs/tutorials/migration-workflow/transform.md

```diff
@@ -30,7 +30,20 @@ In addition Move2Kube also generates the CI/CD pipeline and parameterized versio
 
 ## Transforming using the CLI
 
-1. Run `move2kube transform` in the same directory as the plan file. This will detect the plan file and use it to find the source directory.
+1. Run the transformation in the same directory as the plan file. This will detect the plan file and use it to find the source directory.
+    ```
+    $ move2kube transform
+    ```
+
+    <details markdown="block">
+    <summary markdown="block">
+        Optional: Provide answers to questions using a config file...
+    </summary>
+    If you want to avoid the question answers during transformation, you can use this [config file](https://github.com/konveyor/move2kube-demos/blob/main/samples/enterprise-app/config/m2kconfig.yaml)
+    ```
+    $ move2kube transform --config m2kconfig.yaml
```

Comment:
Probably in the download src phase we should ask them to download the `enterprise-app` directory, so that the config and collect output are available and we can ask users to point to the config in `config` directory